### PR TITLE
Svs fixes

### PR DESCRIPTION
**Pull request by erikogabrielsson**
- Remove jp2 part when encoding jpeg2000.
- Support overview and label pages in svs files. Make pixel_spacing optional.
- Do try to add jpeg_tables if None, and do not add twice.
- Added docstring to Jpeg-class and more descriptive errors.

### CODE REVIEW

File: opentile/common.py [hunk 1]
```diff
@@ -166,9 +166,8 @@ def tile_size(self) -> Size:
     def tiled_size(self) -> Size:
         """The size of the image when tiled."""
         if self.tile_size != Size(0, 0):
-            return self.image_size / self.tile_size
-        else:
-            return Size(1, 1)
+            return self.image_size.ceil_div(self.tile_size)
```
Comment (jesper-molin) [inline opentile/common.py:169]: It feels like ceil_div is the main path of the code, and return Size(1,1) is a special case, so I would flip the if clause and explain why a zero size is counted as 1:

```python
if self.tile_size == Size(0, 0):
    # Explain the reason for this..
    # ...
    return Size(1, 1)

return self.image_size.ceil_div(self.tile_size)
```

File: opentile/common.py [hunk 2]
```diff
@@ -346,10 +345,10 @@ def get_tile(
         """
         tile_point = Point.from_tuple(tile_position)
         frame_index = self._tile_point_to_frame_index(tile_point)
-        frame = self._read_frame(frame_index)
-        if self.page.jpegtables is None:
-            return frame
-        return Jpeg.add_jpeg_tables(frame, self.page.jpegtables)
+        tile = self._read_frame(frame_index)
```
Comment (jesper-molin) [inline opentile/common.py:348]: Can tile be None here? Then return type needs to be optional. And same as above, but the main path without if-clauses and special cases in the if-clause. So from what I can see the, original was good here?

File: opentile/ndpi_tiler.py [hunk 3]
```diff
@@ -615,10 +615,16 @@ def _crop_to_tiles(
...
-        tiles: List[bytes] = self._jpeg.crop_multiple(
-            frame,
-            frame_job.crop_parameters
-        )
+        try:
+            tiles: List[bytes] = self._jpeg.crop_multiple(
+                frame,
+                frame_job.crop_parameters
+            )
+        except JpegCropError:
```
Comment (jesper-molin) [inline opentile/ndpi_tiler.py:623]: This except clause seems pointless to me.

**Review by jesper-molin — CHANGES_REQUESTED** [review]

File: opentile/ndpi_tiler.py (hunk 3 above)
Comment (erikogabrielsson) [reply opentile/ndpi_tiler.py:623]: The point is to give a more detailed exception including what frame that failed, without passing additional data to jpeg crop_multiple().

File: opentile/common.py (hunk 2 above)
Comment (erikogabrielsson) [reply opentile/common.py:348]: Tile can't be None. This arrangement fits better with how add_jpeg_tables() is used elsewhere, where more operations might be run on the tile.

**Review by jesper-molin — APPROVED** [review]

**Pull request merged**